app/javascript/tests/patient/Patient.test.js

```diff
@@ -84,6 +95,15 @@ describe('Patient', () => {
     });
   });
 
+  it('Properly renders contact information section when patient is a minor', () => {
+    const wrapper = shallow(<Patient details={mockPatient5} hoh={mockPatient1} collapse={true} edit_mode={false} jurisdiction_paths={mockJurisdictionPaths} />);
+    const section = wrapper.find('#contact-information');
+    expect(wrapper.find('#contact-information').find('.text-danger').exists()).toBeTruthy();
+    expect(wrapper.find('#contact-information').find('.text-danger').text()).toEqual('Monitoree is a minor. ');
+    expect(section.find('a').exists()).toBeTruthy();
+    expect(section.find('a').last().text()).toEqual(mockPatient1.first_name + ' ' + mockPatient1.middle_name + ' ' + mockPatient1.last_name);
```

Comment:
you can do `find('item-group').find('a')` here instead of doing the last.  I'd also check to make sure the href is correct